cypress.config.js

```diff
@@ -468,6 +470,9 @@ module.exports = defineConfig({
         verifyMoveToCell: (body) => prisonApi.verifyMoveToCell(body),
         stubGetLocationPrefix: ({ agencyId, groupName, response }) =>
           locationsInsidePrisonApi.stubGetLocationPrefix({ agencyId, groupName, response }),
```

Comment:
adding stubGetGroupSearch and then grouping the stubs for locationsInsidePrisonApi together